src/main/java/seedu/address/logic/commands/EditPatientCommand.java

```diff
@@ -0,0 +1,243 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_AGE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_COMMENTS;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PERIOD_OF_STAY;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PHONE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TEMP;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.List;
+import java.util.Optional;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.CollectionUtil;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Age;
+import seedu.address.model.person.Comment;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.PeriodOfStay;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Phone;
+import seedu.address.model.person.Temperature;
+
+/**
+ * Edit a patient in address book with the selected name.
+ * Input of name is case insensitive.
+ */
+public class EditPatientCommand extends Command {
+
+    public static final String COMMAND_WORD = "edit";
```

Comment:
Just a side note, can consider "editPatient" as command word in case we want to have an "editTask" command in future. Up to you! :)

Reply:
oh ya, i forgot. ill change it later